### PR TITLE
fix image

### PR DESCRIPTION
**Pull request by srj31**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -172,12 +172,12 @@ It then returns a `AddCommand` back to `UniFyParser`, initialized with the `Task
 
 ##### Usage Scenario
 
-The following demonstrates a usage scenario where the user wants to add a task with name CS2103, date 2021-10-10, time 23:59, priority level LOW and tag Important
+The following demonstrates a usage scenario where the user wants to add a task with name Test, date 2021-10-10, time 23:59 (default time), priority level LOW(default priority level)) and tag CS2103
```
Comment (lwj1711) [inline docs/DeveloperGuide.md:175]: Additional ')' here

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -629,14 +635,32 @@ testers are expected to do more *exploratory* testing.
...
 
    1. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
+3. _{ more test cases …​ }_
+
+### Adding a task
+
+1. Deleting a task while all tasks are being shown
```
Comment (lwj1711) [inline docs/DeveloperGuide.md:649]: 'Deleting' should be changed to 'Adding' right

Comment (lwj1711) [reply docs/DeveloperGuide.md:649]: Also think we can omit the 'while all tasks are being shown part'

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -629,14 +635,32 @@ testers are expected to do more *exploratory* testing.
...
+
+    1. Prerequisites: List all tasks using the `list` command. At least one task in the list.
```
Comment (lwj1711) [inline docs/DeveloperGuide.md:651]: I don't think we need this prerequisite for 'add' right

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (srj31) [reply docs/DeveloperGuide.md:175]: thanks for correcting the typo

**Comment by lwj1711** [comment]
Looks great

**Pull request merged**